.github/workflows/license-check.yml

```
@@ -4,15 +4,77 @@ on:
+            worker/go.sum
+      - name: Web changes
+        id: web
+        uses: step-security/changed-files@3dbe17c78367e7d60f00d78ae6781a35be47b4a1 # v45.0.1
+        with:
+          files: |
+            web/package.json
+            web/yarn.lock
+
   server-license-check:
     name: Server License Check
+    if: needs.prepare.outputs.server == 'true' || github.event_name == 'workflow_dispatch'
```

actionlint (1.7.4)

property "prepare" is not defined in object type {}

(expression)